### PR TITLE
Add XML documentation for IFormFileCollection

### PR DESCRIPTION
**Pull request by Marusyk**
Issue https://github.com/aspnet/Home/issues/3528

Please review

### CODE REVIEW

**Comment by Tratcher** [comment]
Thanks for the update.

File: src/Microsoft.AspNetCore.Http.Features/IFormFileCollection.cs [hunk 1]
```diff
@@ -10,10 +10,33 @@ namespace Microsoft.AspNetCore.Http
...
+        /// </summary>
+        /// <param name="name">The name of the file to get.</param>
+        /// <returns>
+        ///	The requested file, or null if it is not present.
+        /// </returns>
         IFormFile GetFile(string name);
 
+        /// <summary>
+        ///     Gets an <see cref="IReadOnlyList{T}" /> containing the files of the
+        ///     <see cref="IFormFileCollection" /> with the specified name.
```
Comment (JamesNK) [inline src/Microsoft.AspNetCore.Http.Features/IFormFileCollection.cs]: If you can have multiple files with the same name, what does GetFile and the indexer do when there are multiple? Return the first one/last one/throw an error?

Comment (Tratcher) [reply src/Microsoft.AspNetCore.Http.Features/IFormFileCollection.cs]: Returns the first.
https://github.com/aspnet/HttpAbstractions/blob/cbed7393a21483cf25cda77dea8b7cda2b51751f/src/Microsoft.AspNetCore.Http/Internal/FormFileCollection.cs#L20-L26

Comment (JamesNK) [reply src/Microsoft.AspNetCore.Http.Features/IFormFileCollection.cs]: Ok! Could be worth adding to the XML docs, but that's probably what I'd expect to happen in the rare situation it comes up.

Comment (Tratcher) [reply src/Microsoft.AspNetCore.Http.Features/IFormFileCollection.cs]: up-for-grabs 😁

Comment (Marusyk) [reply src/Microsoft.AspNetCore.Http.Features/IFormFileCollection.cs]: Should I prepare a new PR for this?

Comment (Eilon) [reply src/Microsoft.AspNetCore.Http.Features/IFormFileCollection.cs]: @Marusyk that would be great if you're willing to!

**Comment by Marusyk** [comment]
Related #1041

File: src/Microsoft.AspNetCore.Http.Features/IFormFileCollection.cs (hunk 1 above)
Comment (Marusyk) [reply src/Microsoft.AspNetCore.Http.Features/IFormFileCollection.cs]: https://github.com/aspnet/HttpAbstractions/pull/1041

**Pull request merged**